events/workshops-ebi-micro-may-2023.html

```diff
@@ -0,0 +1,73 @@
+---
+layout: news-events
+filename: events
+title: May 2023 OME workshops
+main-blurb: The OME Team gave two workshops on OMERO, OME-NGFF and IDR as a part of the Microscopy data analysis course at EMBL-EBI.
```

Comment:
I won't use the term workshops here
Practicals will be more suitable to match the terminology used in the course